CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+
+## Code of Conduct
+
+We expect all contributors to adhere to our Code of Conduct, which promotes a friendly and inclusive environment for everyone. Please make sure to read and follow it in all your interactions within the project.
+
+We are using the following tech:
+
+HTML5
+
+We are using Tailwind for styling
+
+## How to Contribute
+
+1. **Fork the Repository**: Click the "Fork" button on the top right corner of this repository to create your copy of the project.
+
+2. **Clone the Repository**: Clone your forked repository to your local machine using the following command:
+
+   ```bash
+   git clone https://github.com/your-username/jslovers.git
+   ```
+
+3. **Create a Branch**: Before making any changes, create a new branch to work on. Please use a descriptive name for your branch.
+
+   ```bash
+   git checkout -b feature/your-feature-name
+   ```
+
+4. **Make Your Changes**: Make your desired changes or additions to the codebase.
+
+5. **Test Your Changes**: Ensure that your code is thoroughly tested and doesn't break any existing functionality. Also, make sure your code follows our coding standards.
+
+6. **Commit Your Changes**: Commit your changes with a descriptive commit message. Please follow the [Conventional Commits](https://www.conventionalcommits.org/) format if possible.
+
+   ```bash
+   git commit -m "feat: Add new feature"
+   ```
+
+7. **Push Your Changes**: Push your branch to your forked repository.
+
+   ```bash
+   git push origin feature/your-feature-name
+   ```
+
+8. **Create a Pull Request**: Open a pull request from your fork to this main repository. Please provide a detailed description of your changes and reference any relevant issues if applicable.
+
+9. **Review and Discussion**: Your pull request will undergo review by the maintainers and other contributors. Be prepared to address feedback and make necessary changes.
+
+10. **Merge**: Once your pull request is approved, it will be merged into the main branch.
+
+## Coding Standards
+
+Please follow the coding standards and guidelines specific to the "jslovers" project. If you're unsure about any coding standards, you can check the project's `CONTRIBUTING.md` or ask for clarification in your pull request or by reaching out to maintainers.
+
+## Issues
+
+You can contribute not only through code but also by helping to identify and resolve issues. Feel free to browse our issue tracker, pick an issue that interests you, and leave a comment that you're working on it.
+
+## Communication
+
+If you have questions or need assistance, you can reach out to the project maintainers through GitHub issues or other communication channels, such as a dedicated Discord server or Slack workspace (if applicable).
+
+## Attribution
+
+Contributors will be credited for their contributions in the project's README or a dedicated contributors file.
```

Comment:
We don't have any contribution files. Nor, we are going to add in the Project's readme as of now. Please as of now, remove this line.

The contribution will be visible on the GitHub.